docs/benchmarks/data-caching.md

```diff
@@ -73,43 +58,47 @@ The client can simultaneously use multiple servers, one server with several ip a
     IP_ADDRESS_VM2, 11211
     ...
 
-You can use the `vim` command for modifying this file inside the container.
+
+
+To start the client container use the following command:
+
+    $ docker run -idt --name dc-client --net host -v PATH_TO_DOCKER_SERVERS_FOLDER:/usr/src/memcached/memcached_client/docker_servers/ cloudsuite/data-caching:client
+
+Please note that the command mounts the folder containing the 'docker_servers.txt' file instead of mounting only the file. This way, further changes to the docker_servers.txt file in the host will be reflected inside of the container. 
```

Comment:
Maybe directly mapping the `docker_servers.txt` file into the container is more direct than mapping the folder. What do you think?

Reply:
I don't remember ever modifying anything besides the `docker_servers.txt` file, so yes I think we should just map that one file.

Reply:
I recommend mapping a folder because this way, changes to the file in the host after starting the container will be reflected inside the container. Otherwise, if a user changed the file in the host, the inode of the file changes and the container will have a stale file.